packaging/greenboot/green.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -ex
+
+HEALTH=healthy
+
+if [ ! -f /run/ostree-booted ]; then
+    echo "System is not booted with ostree"
+    exit 0
+fi
+
+mkdir -p /var/lib/microshift-backups
+
+boot=$(tr -d '-' < /proc/sys/kernel/random/boot_id)
```

Comment:
Why removing `-` from the boot ID? We may need in future to correlate it to the actual value. 
Let's keep the ID unchanged if it's not a problem?

Reply:
I'm removing it so it's the same format as kept by `journalctl --list-boots` (it also removes `-`).
If someone would want to match backup to specific boot, they'd probably use mentioned command